wrappers/_shared.nix

```diff
@@ -42,18 +43,34 @@ in
 
   config = mkMerge [
     # Make our lib available to the host modules
-    { nixvim.helpers = lib.mkDefault (import ../lib/helpers.nix { inherit pkgs lib; }); }
```

Comment:
@MattSturgeon wasn't this necessary?

Reply:
Technically this is unchanged, the import is moved to the `let` block above.

I'd prefer to keep the import here and have `inherit (config.nixvim) helpers` in the let-block instead, though.